devices/Paexo/test-application/paexo-test-application.cpp

```diff
@@ -23,44 +29,94 @@ int main() {
     std::string outPort = "/Paexo/WearableActuatorsCommand/output:o";
 
-    if (!yarp::os::Network::exists(inPort))
-    {
-        yError() << "Port " << inPort << " does not exists";
-        return -1;
-    }
+    std::vector<std::string> PaexoActuators = {"Paexo::motor::LeftMotor", "Paexo::motor::RightMotor"};
 
-    BufferedPort<wearable::msg::WearableActuatorCommand> port;
-    if(!port.open(outPort))
-    {
-        yError() << "Failed to open port " << outPort;
-        return -1;
-    }
+    std::vector<std::string> PaexoActuatorPortNames = {"/Paexo/motor/LeftMotor"};
+    std::vector<std::unique_ptr<YarpBufferedPort>> PaexoActuatorPorts;
+
+//    if (!yarp::os::Network::exists(inPort))
+//    {
+//        yError() << "Port " << inPort << " does not exists";
+//        return -1;
+//    }
 
-    if (!yarp::os::Network::connect(outPort,inPort))
+//    BufferedPort<wearable::msg::WearableActuatorCommand> port;
+//    if(!port.open(outPort))
+//    {
+//        yError() << "Failed to open port " << outPort;
+//        return -1;
+//    }
+
+//    if (!yarp::os::Network::connect(outPort,inPort))
+//    {
+//        yError() << "Failed to connect " << outPort << " to " << inPort;
+//        return -1;
+//    }
+
+    // Yarp buffered ports for motor control
+    for (const auto& portName : PaexoActuatorPortNames)
     {
-        yError() << "Failed to connect " << outPort << " to " << inPort;
-        return -1;
+        std::unique_ptr<YarpBufferedPort> port = std::make_unique<YarpBufferedPort>();
+
+        if (!port->open(portName + ":o"))
+        {
+            yError() << "Failed to open port " << (portName + ":o");
+            return -1;
+        }
+
+        if (!yarp::os::Network::connect(portName + ":o", portName + ":i"))
+        {
+            yError() << "Failed to connect ports " << (portName + ":o") << " and " << (portName + ":i");
+            return -1;
+        }
+
+        PaexoActuatorPorts.push_back(std::move(port));
     }
 
+
     while (true) {
 
-        wearable::msg::WearableActuatorCommand& wearableActuatorCommand = port.prepare();
+//        for (const auto& actuator : PaexoActuators)
+//        {
+//            wearable::msg::WearableActuatorCommand& wearableActuatorCommand = port.prepare();
+
+//            // Add wearable actuator command
+//            wearableActuatorCommand.info.name = actuator;
+//            wearableActuatorCommand.info.type = wearable::msg::ActuatorType::MOTOR;
+//            wearableActuatorCommand.info.status = wearable::msg::ActuatorStatus::OK;
+
+//            wearableActuatorCommand.duration = 10;
+//            wearableActuatorCommand.value = 40;
+
+//            yInfo() << "Command " << wearableActuatorCommand.info.name << " to position "
+//                    <<  wearableActuatorCommand.value << " deg";
+
+//            // Send the actuator command to the output port
+//            port.write(true);
+
+//            Time::delay(2);
+//        }
```

Comment:
same as above

Reply:
Done in https://github.com/robotology/wearables/pull/124/commits/faefcae8f616d0b1a912f78e50a7340c42e037a0